lib/streamlit/connections/base_connection.py

```diff
@@ -25,10 +25,7 @@
 class BaseConnection(ABC, Generic[T]):
```

Comment:
this is definitely out of scope, but we could have more descriptive name for TypeVar, something like `RawConnectionT`

Reply:
Will do